### DISCUSSION TITLE
Update 20240310 shows shared-data

### DISCUSSION THREAD

**Discussion by AndyFx12** [rern/rAudio#1621] [Fixed Issues]
- Latest update shows /mnt/MPD/NAS/data - a folder used by "Server rAudio". So it should be hidden.
- update seems to be significant slower on browsing my library, hosted on QNAP/NAS.
Please make "show hidden/empty things" optional.

regards, Andy

**Comment by markblinkhorn** [comment]
Agreed, library browsing does seem significantly slower. I thought it had hung up at first and had to wait for it to catch up with my repeated finger stabs on various folders. My files are hosted on a Synology iSCSI volume attached to Windows Server file share - so I suspect it is the size of the library rather than the underlying connection technology. I have approx 37000 items in my library.

**Reply by rern** [reply]
Please try the UPDATE.

**Reply by AndyFx12** [reply]
Update-branch don´t fix this problem - it's still slower and shows shared-data. 
In the morning (german time, 06:00am), there was an update-version, which fixed both problems - but current version doesn´t.

**Reply by rern** [reply]
@AndyFx12,

It's released as today's update,

Another thing about Spotify, do you have [the volume control issue](https://github.com/rern/rAudio/discussions/1528)?

**Reply by AndyFx12** [reply]
@rern 
Today's update has same behaviour as update from 20240310 - shared data is still visible. And it's slow "as hell" - compared to previous versions.

And yes - spotify volume control via client is broken. Seems to work permanently with maximum volume.

**Reply by rern** [reply]
Please try the UPDATE again.

**Reply by AndyFx12** [reply]
+ shared-data folder is hidden, now
- speed is far away from "before". I have 2 identical RPi3 with same data. Navigation to a folder with many files (~1000) is similar fast. But navigation from root to list of ~30 subfolders takes 4-5-times longer. Before: nearly at once. After: 1-2 seconds. Same by navigation one folder up.
regards, Andy

**Reply by AndyFx12** [reply]
@rern 
Current update is working fine, again. Thanks!

**Reply by rern** [reply]
For the Spotify volume issue

Please try:
```sh
sed -i -E 's/(hw:.),./\1/' /etc/spotifyd.conf
systemctl restart spotifyd
```

**Reply by AndyFx12** [reply]
This command fixes this issue on my setup. 
hw:0,0 changed to hw:0 at spotifyd.conf
Thanks!

**Comment by kaufmach** [comment]
I do confirm "slow library browsing" - indeed _very_ slow although nothing else changed except update. Library connected to USB hard drive.

**Comment by AndyFx12** [comment]
Fixed at Update-branch.
